OnCampusTweaks.user.js

```diff
@@ -53,9 +51,76 @@ function hideCompletedTasks() {
         console.log("OnCampusTweaks: Hid completed tasks");
     }
 }
-
+function AlwaysShowGrades() {
```

Comment:
also, in javascript it's best to use camelCase for functions. When you capitalize a function, it should be a constructor, according to common conventions.  ([source](https://stackoverflow.com/a/1564489))

Reply:
So noted